myst build is failing

The nightly build (and all other builds) have been failing since January 11 with what appears to be a node issue. E.g. this build log excerpt:
```
Run cd .
  cd .
  myst build --execute --html
  shell: /usr/bin/bash -leo pipefail {0}
  env:
    BASE_URL: 
    CI_REPOSITORY_SLUG: projectpythia-pythia-foundations
    CI_REPOSITORY_OWNER: ProjectPythia
    CI_REPOSITORY_OWNER_SLUG: projectpythia
    CI_REPOSITORY_NAME: pythia-foundations
    CI_REPOSITORY_NAME_SLUG: pythia-foundations
    CI_REPOSITORY: ProjectPythia/pythia-foundations
    CI_REF_SLUG: refs-heads-main
    CI_REF_NAME: main
    CI_REF_NAME_SLUG: main
    CI_REF: refs/heads/main
    CI_ACTION_REF_NAME: main
    CI_ACTION_REF_NAME_SLUG: main
    CI_HEAD_REF: 
    CI_BASE_REF: 
    CI_SHA_SHORT: 2635ce61
    CI_SHA: 2635ce61fd4c1154d09f41c0fa3a7569736e615c
    CI_ACTOR: brian-rose
    CI_EVENT_NAME: schedule
    CI_RUN_ID: 20905031327
    CI_RUN_NUMBER: 1527
    CI_WORKFLOW: nightly-build
    CI_ACTION: get-env
    MAMBA_ROOT_PREFIX: /home/runner/micromamba
    MAMBA_EXE: /home/runner/micromamba-bin/micromamba
    CONDARC: /home/runner/work/_temp/setup-micromamba/.condarc
    PRIVATE_KEY: 
    PRIVATE_KEY_ID: 
    ARM_USERNAME: 
    ARM_PASSWORD: 
    AQS_USERNAME: 
    AQS_KEY: 
    EARTHDATA_USERNAME: 
    EARTHDATA_PASSWORD: 
    SECRETS_VARS: {
    "github_token": "***"
  }
Traceback (most recent call last):
  File "/home/runner/micromamba/envs/pythia-book-dev/bin/myst", line 10, in <module>
    sys.exit(main())
             ~~~~^^
  File "/home/runner/micromamba/envs/pythia-book-dev/lib/python3.14/site-packages/mystmd_py/main.py", line 32, in main
    f"MyST requires node 18, 20, or 22+; you are running node {version[1:3]}.\n\n"
                                                               ^^^^^^^
NameError: name 'version' is not defined. Did you mean: '_version'?
Error: Process completed with exit code 1.
```

--- a/environment.yml
+++ b/environment.yml
@@ -19,5 +19,4 @@ dependencies:
   - python-graphviz
   - graphviz
   - mystmd
-  - nodejs
   - jupyterlab-myst

